armoryengine/ArmoryUtils.py

```diff
@@ -131,6 +131,9 @@
 parser.add_option("--coverage_output_dir", dest="coverageOutputDir", default=None, type="str", help="Unit Test Argument - Do not consume")
 parser.add_option("--coverage_include", dest="coverageInclude", default=None, type="str", help="Unit Test Argument - Do not consume")
 
+# signassert.py arguments
+parser.add_option("-u", "--signer", dest="signer", type="str", help="Wallet code and address in form code/address")
```

Comment:
I'm not sure I like "code." "ID" seems more appropriate and matches what's in the Armory GUI.